mk/blktap.spec.in

```diff
@@ -83,7 +83,13 @@ mkdir -p %{buildroot}%{_localstatedir}/log/blktap
 %{_includedir}/blktap/*
 
 %post
-[ ! -x /sbin/chkconfig ] || chkconfig --add tapback
+%systemd_post tapback.service
 [ ! -x /sbin/chkconfig ] || chkconfig --add thinprovd
 
+%preun
+%systemd_preun tapback.service
+
+%postun
+%systemd_postun tapback.service
+
```

Comment:
You should add the correct dependencies:
`Requires(post): systemd
Requires(preun): systemd
Requires(postun): systemd
BuildRequires: systemd`